webapp/packages/core-blocks/src/FormControls/InputFiles.tsx

```diff
@@ -195,28 +172,28 @@ export const InputFiles: InputFilesType = observer(
 
     const files = Array.from(value ?? []);
 
-    return styled(styles)(
-      <field className={s(sizeStyles, { ...layoutProps }, className)}>
-        <field-label title={labelTooltip || rest.title}>
+    return (
+      <div className={s(styles, { ...layoutProps, field: true }, className)}>
+        <label title={labelTooltip || rest.title} className={s(styles, { fieldLabel: true })}>
           {children}
           {required && ' *'}
-        </field-label>
-        <input-container>
-          <UploadArea ref={ref} {...rest} name={name} value={value} {...use({ mod })} required={required} onChange={handleChange}>
+        </label>
+        <div className={s(styles, { inputContainer: true })}>
+          <UploadArea ref={ref} {...rest} name={name} value={value} required={required} onChange={handleChange}>
             <Button icon="/icons/import.svg" tag="div" loading={loading} mod={['outlined']}>
               {translate(rest.multiple ? 'ui_upload_files' : 'ui_upload_file')}
             </Button>
           </UploadArea>
           {!hideTags && (
-            <Tags>
+            <Tags className={s(styles, { tags: true })}>
               {files.map((file, i) => (
                 <Tag key={file.name} id={i} label={file.name} onRemove={removeFile} />
               ))}
             </Tags>
           )}
-        </input-container>
-        {description && <field-description>{description}</field-description>}
-      </field>,
+        </div>
+        {description && <div className={s(styles, { fieldDescription: true, invalid: error })}>{description}</div>}
```

Comment:
probably we need to move it to a separate component, `FieldDescription` and use it instead so we don't need to pass `fieldDescription` classes every time